examples/azure_ad/group_sync.py

```diff
@@ -0,0 +1,33 @@
+"""Additional functions to process an Azure user."""
+import logging
+from django.contrib.auth.models import Group
+
+logger = logging.getLogger("auth.Azure")
```

Comment:
Is this a pre-existing logger that we're tying into, or should this be something like `logging.getLogger(__name__)` instead?

Reply:
This is mimicking the logger that was in the Okta example.